Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -5,6 +5,8 @@
 #include "DolphinQt2/GameList/GameListModel.h"
 #include "DolphinQt2/Resources.h"
 
+static const QSize GAMECUBE_BANNER_SIZE(96, 32);
```

Comment:
If you want the full listing of divergences between C and C++, check out appendix C in the C++ standard. It also lists the rationales behind them as well =)